cmd/finalize/finalize.go

```diff
@@ -23,14 +23,14 @@ type Message struct {
 func main() {
 	config := NewConfig()
 	mq := broker.New(config.Broker)
-	db, err := postgres.NewDB(config.Postgres)
+	dbs, err := postgres.NewDB(config.Postgres)
 	if err != nil {
 		log.Println("err:", err)
 	}
 
+	defer dbs.Close()
```

Comment:
Unnecessary renaming
```suggestion
	defer db.Close()
```